src/plugins/intel_cpu/src/nodes/mvn.cpp

```diff
@@ -1829,7 +1829,7 @@ void MVN::initSupportedPrimitiveDescriptors() {
 
     ov::element::Type inputPrecision = getOriginalInputPrecisionAtPort(0);
     ov::element::Type outputPrecision = getOriginalOutputPrecisionAtPort(0);
-    if (!mayiuse(avx512_core)) {
+    if (!mayiuse(avx512_core) && !mayiuse(avx2_vnni_2)) {
```

Comment:
General suggestion. We have intel_cpu::hasHardwareSupport(...) utility. 
Can we try to reuse it to replace such code invocations? Smt like:
```
if (!hasHardwareSupport(outputPrecision))
    outputPrecision = ov::element::f32
```

Reply:
have changed this and some other places in CPU Nodes which also used this kind of way.